self_teach_challenge_1.1/script.js

```diff
@@ -21,18 +71,23 @@ const COUNTDOWN = () => {
     interval = setInterval(() => {
         if (countdownSeconds > 0) {
             countdownSeconds = countdownSeconds - 1;
+            minutesInput.setAttribute('value', (Math.floor(Number(countdownSeconds/60))))
+            secondsInput.setAttribute('value',countdownSeconds%60)
+            minutesInput.value = Math.floor(Number(countdownSeconds/60));
+            secondsInput.value = Number(countdownSeconds%60);
+            oneNumber();
             console.log(countdownSeconds)
+            console.log(secondsInput.value.length)
```

Comment:
Do: Get into the practice of not committing `console.log`s, otherwise it might end up in your production code (and leak sensitive information)